code/__DEFINES/multi_z.dm

```diff
@@ -1,3 +1,5 @@
 
-//Amount of units between station levels.
+/// Amount of units between station levels.
 #define MULTI_Z_DISTANCE 5
+/// Multiplier to MULTI_Z_DISTANCE in the case of explosions
+#define MULTI_Z_EXPLODE_FACTOR 0.4
```

Comment:
Keep in mind that explosions between floors are already massively reduced by the  cost of vertical travel

Reply:
No this is increasing the explosion between zs, otherwise almost nothing happens